commit c0381d2b596365f33bb732efa269572a713b3092 (refs/changes/82/1882/3)
Author: David Pacheco <dap@joyent.com>
Date:   2017-05-02T06:58:50-07:00 (2 years, 5 months ago)
    
    MANTA-3241 clarify manta-genazconfig docs
    Reviewed by: Brian Bennett <brian.bennett@joyent.com>
    Reviewed by: Richard Bradley <richard.bradley@joyent.com>
    Approved by: Brian Bennett <brian.bennett@joyent.com>

diff --git a/README.md b/README.md
index 001c175..f0af025 100644
--- a/README.md
+++ b/README.md
@@ -70,8 +70,8 @@ This tool itself has a configuration file that describes:
 * the list of regions (each region used for a separate Manta deployment)
 * for each region, the list of availability zones (AZs)
 * for each AZ, the name of the "building" in the Device42 database, the list of
-  rack names in the Device42 database that should be part of this Manta
-  deployment, and an optional CNAPI endpoint for the AZ
+  rack names in the Device42 database _that should be part of this Manta
+  deployment_, and an optional CNAPI endpoint for the AZ
 * optionally, for each AZ, a description of the intended networking
   configuration (see the
   [Additional `gen-manta-net` configuration](#additional-gen-manta-net-configuration)
@@ -108,6 +108,13 @@ deployment using two racks' worth of servers per DC:
         }
     }
 
+Note that `"d42racks"` is the list of racks containing servers that you want to
+use for Manta.  This tool will generate configuration causing Manta services to
+deployed to all servers in those racks having a recognized hardware type.  You
+should not specify any racks here containing servers you don't want to use for
+Manta.
+
+
 ## Synopsis
 
 First, clone and build:
@@ -158,7 +165,11 @@ subcommand the following configuration additions must be made.
 
 * In the list of AZs for a region, a "networking" object must be added that
   contains NAPI network and NIC mapping information for the "admin", "manta",
-  and "mantanat" network. For example:
+  and "marlin" networks.  Note that the `"marlin"` network is usually called
+  `"mantanat"` in Triton, and it usually has NIC tag `"mantanat"` as well, but
+  `manta-net.sh` calls it `"marlin"`, so that's what's used here.
+
+  For example:
 
       ...
       }, {
@@ -180,17 +191,20 @@ subcommand the following configuration additions must be made.
               "manta": {
                   "network": {
                   ...
+              "marlin": {
+                  "network": {
+                  ...
       ...
 
-The "nic_mapping" value is assumed to be an aggregation name unless the
+The `"nic_mapping"` value is assumed to be an aggregation name unless the
 special value of `<mac>` is used. When `<mac>` is used, the resulting
 configuration file will contain placeholder values (`$server_uuid-MAC"`) that
 need to be replaced with the appropriate MAC address for that server's "manta"
 and "mantanat" network.
 
-The per-network "nic_mapping" value is also assumed to be the same value across
-all servers in the AZ. That is, if you require a subset of storage servers to
-have a different aggregation tagged for "manta" usage, the resulting
+The per-network `"nic_mapping"` value is also assumed to be the same value
+across all servers in the AZ. That is, if you require a subset of storage
+servers to have a different aggregation tagged for "manta" usage, the resulting
 configuration file must be edited to reflect this.
 
 
